include/deal.II/differentiation/ad/ad_helpers.h

```diff
@@ -2837,8 +2837,7 @@ namespace Differentiation
        * @note For taped AD numbers, this operation is only valid in recording mode.
        */
       template <typename ExtractorType>
-      typename internal::Extractor<dim,
-                                   ExtractorType>::template tensor_type<ad_type>
+      auto
       get_sensitive_variables(const ExtractorType &extractor) const;
```

Comment:
Can we use a more descriptive return type?

Reply:
I found an alternative that the compiler accepts.